api/BarProp.json

```
@@ -96,6 +96,54 @@
             "deprecated": false
           }
         }
+      },
+      "visible_popup": {
+        "__compat": {
+          "description": "<code>visible</code> represents popup or not",
```

This is really challenging to explain in just a couple of words! I'm going to suggest this. It doesn't tell the whole story, but I think it gets us pretty close. Open to further suggestions, if you like.

```suggestion
          "description": "Returns <code>true</code> for non-popup windows",
```